protocol/03_0_encoding.md

```diff
@@ -9,47 +9,43 @@
 A message is made up of two JSON strings, one for the header and the other for the data.
 
 ### Header
-The header is always 64 _UTF-8_ characters long. That means the payload should be exactly 64 bytes long, encoded big endian. 
+The header is always 53 _UTF-8_ characters long. That means the payload should be exactly 53 bytes long, encoded big endian. 
 
-This allows the receiving side to first fill an input buffer with the data for exactly one JSON string, and extract further information from there. The content of the header is `{"Header":{"Length":"01234","CRC32":"0123456789","type_id":"1"}}`, or formatted to look more legible:
+This allows the receiving side to first fill an input buffer with the data for exactly one JSON string, and extract further information from there. The content of the header is `{"length":"01234","crc32":"0123456789","type_id":"1"}`, or formatted to look more legible:
 ```json
 {
-  "Header": {
-      "Length": "01234",
-      "CRC32" : "0123456789",
-      "type_id": "1"
-    }
+  "length": "01234",
+  "crc32" : "0123456789",
+  "type_id": "1"
 }
 ```
 
 The information here is used to receive the data JSON.
 
 | Name   | Description                                             | Target Type         |
 |--------|---------------------------------------------------------|---------------------|
-| Length | Length of the 2nd JSON string (Data) in bytes.          | `uint16` or `int32` | 
-| CRC32  | CRC32 checksum over the buffer of the 2nd JSON string.  | `uint32` or `int64` |
+| length | Length of the 2nd JSON string (Data) in bytes.          | `uint16` or `int32` | 
+| crc32  | CRC32 checksum over the buffer of the 2nd JSON string.  | `uint32` or `int53` |
```

Comment:
int53?

Reply:
I CTRL+F replaced 64 bytes with 53 bytes because I reduced the header size, oopsie 😆